src/components/tools/Location/StaticMap.svelte

```diff
@@ -11,15 +14,13 @@
   export let style = "mapbox/streets-v11";
   export let padding = 50;
   export let zoom = 8;
-  export let alt = "location map";
 
+  const { accessToken } = mapboxgl;
   let img;
```

Comment:
Because you're now using the  `ImageLoader` component I think you can get rid of these unused variables (img, loading, error) as well as remove the `load()` function and the `if` statement that calls it below.